### DISCUSSION TITLE
Suggestion for PODS Block - Lazy Blocks Plugin

### DISCUSSION THREAD

**Discussion by poirazis** [pods-framework/pods#6803] [Ideas]
## Problem

A more powerful way to build blocks with advanced logic for PODS

## Proposed Solution

I came across this plugin, https://lazyblocks.com/.
I wanted to bring this to your attention. I think such a capability built into PODS would make PODS framework much much more powerful and popular.

**Comment by sc0ttkclark** [comment]
I'm interested in creating a UI for this. Pods 2.8 introduces a new Blocks PHP API and Blocks JS API for Pods, which we use for our own blocks. I hope to expose the capabilities there with a new builder screen that's a more restricted version of the normal Edit Pod screen. That'll take quite a bit of thought and preparation but it's on my list.
